### PR TITLE
[IOPID-1399] Add opex dashboard for "api_backend"

### PR DESCRIPTION
**Pull request by lucacavallaro**
#### List of Changes

1. Add Azure Dashboard to monitor `api_backend.yaml` endpoints

#### Motivation and Context

This dashboard enables us to monitor core endpoints like (`/profiles` that was recently modified for the unique e-mail enforcement feature and `/messages`)

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by gquadrati — CHANGES_REQUESTED** [review]
We need to add GH actions

File: api_backend.yaml
Comment (arcogabbo) [inline api_backend.yaml:1]: why are we doing this changes? the referenced spec in `config.yaml` is on `generated` directory

File: .opex/api_backend/env/prod/config.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+event_occurrences: 4 # After 5*4=20 min
+data_source: /subscriptions/ec285037-c673-4f58-b594-d7c480da4e8b/resourceGroups/io-p-rg-external/providers/Microsoft.Network/applicationGateways/io-p-appgateway
+action_groups:
+  - /subscriptions/EC285037-C673-4F58-B594-D7C480DA4E8B/resourceGroups/io-p-rg-common/providers/microsoft.insights/actionGroups/iopquarantineerror
+overrides:
+  hosts:
+    - app-backend.io.italia.it
+    - api-app.io.pagopa.it
+  endpoints:
+    /profile:
```
Comment (arcogabbo) [inline .opex/api_backend/env/prod/config.yaml:16]: this should fix the error in the pipeline
```suggestion
    /api/v1/profile:
```

**Review by arcogabbo — CHANGES_REQUESTED** [review]

File: api_backend.yaml
Comment (lucacavallaro) [reply api_backend.yaml:1]: the file reference in `config.yaml` is `api_backend.yaml` from the root, the `schema` property is not supported on `openapi 2.0`

File: api_backend.yaml [hunk 2]
```diff
@@ -14,9 +14,8 @@ paths:
     x-swagger-router-controller: ServicesController
     parameters:
       - in: path
+        type: string
         name: service_id
-        schema:
-          $ref: "#/definitions/PathTraversalSafePathParam"
         required: true
```
[the remark this answers is not in the document]
Comment (gquadrati) [reply api_backend.yaml:19]: Actually, the openapi in master branch is not valid, so I close this comment.

**Comment by arcogabbo** [comment]
`api_backend.yaml` invalid spec was fixed in #1099 @lucacavallaro

**Comment by gquadrati** [comment]
@lucacavallaro, as mentioned by @arcogabbo, we have addressed the specifications. 
Once conflicts are resolved, we can proceed with the merge.

**Review by gquadrati — APPROVED** [review]

**Pull request merged**